test/test_sam.py

```diff
@@ -2272,13 +2272,15 @@ def testOverlappingCoverageEightyPercent(self):
                 (350, 100),
             )
         ):
-            rr.add(
-                AlignedRead(
-                    query_name=str(count),
-                    reference_start=start,
-                    reference_length=length,
-                )
-            ),
+            (
+                rr.add(
+                    AlignedRead(
+                        query_name=str(count),
+                        reference_start=start,
+                        reference_length=length,
+                    )
+                ),
+            )
```

Comment:
This change looks odd. The code is exactly the same, but it's putting the result of `rr.add` into a tuple. Why?

Reply:
Because there was a trailing comma in the original.